spec/support/exception_helpers.rb

```diff
@@ -6,4 +6,20 @@ def actual_exception
   rescue => e # rubocop:disable Style/RescueStandardError
     e
   end
+
+  # rubocop:disable Metrics/MethodLength
+  def actual_chained_exception
+    1 / 0
+  rescue ZeroDivisionError
+    begin
+      File.open('gotcha')
```

Comment:
Sending happy thoughts to the day where some weird tool creates `gotcha` files and this mysteriously doesn't work anymore 😆